### ISSUE TITLE
Version mismatches with recent saturn & giraffe

### ISSUE THREAD

**Issue by pkese** [Nhowka/Elmish.Bridge#34]
It seems that everything is moving ahead (including new versions of Saturn, Giraffe and Fable 3) and people are updating their software so things are starting to break.

Many recent starter projects, like https://github.com/Zaid-Ajaj/SAFE.Simplified will break dependencies that come with current version of Elmish.Bridge, because other packages are pulling in a more recent version of Giraffe:
```
Unhandled exception. System.IO.FileLoadException:
Could not load file or assembly 'Giraffe, Version=3.6.0.0, Culture=neutral, PublicKeyToken=null'.
The located assembly's manifest definition does not match the assembly reference. (0x80131040)
File name: 'Giraffe, Version=3.6.0.0, Culture=neutral, PublicKeyToken=null'
   at Elmish.Bridge.Giraffe.server(String endpoint, FSharpFunc`2 inboxCreator)
   at <StartupCode$Server>.$Program.main@()
```
Some people have already made necessary code fixes: https://github.com/Nhowka/Elmish.Bridge/network

Is there any chance of spinning a new version on Nuget?

**Comment by Nhowka** [comment]
Yes, thanks for bringing my attention to it! There is also interesting experiments happening there that might be worth looking to fix more user cases. I'll update the versions, do some tests and publish a new version in the morning.

**Comment by Nhowka** [comment]
I just published the version [5.0.0-rc-2](https://www.nuget.org/packages/Elmish.Bridge.Giraffe/5.0.0-rc-2) for Elmish.Bridge.Giraffe, can you check if it is working ok? I also changed some stuff on the server to change the Mailbox implementation to one using the Elmish Program.

**Comment by pkese** [comment]
Works like a charm. Thanks a lot.
